fairlearn/reductions/_exponentiated_gradient/exponentiated_gradient.py

```diff
@@ -35,37 +35,43 @@ class ExponentiatedGradient(BaseEstimator, MetaEstimatorMixin):
         violation is at most :code:`2*(eps+best_gap)`
     :type eps: float
 
-    :param T: Maximum number of iterations
-    :type T: int
+    :param max_iterations: Maximum number of iterations
+    :type max_iterations: int
 
```

Comment:
if we are renaming this, let's follow [sklearn](https://scikit-learn.org/stable/glossary.html#parameters): `max_iter`